Altis_Life.Altis/core/vehicle/fn_openInventory.sqf

```diff
@@ -22,7 +23,7 @@ if (_vehicle isKindOf "Box_IND_Grenades_F" || _vehicle isKindOf "B_supplyCrate_F
     ctrlSetText[3501,format [(localize "STR_MISC_VehStorage")+ " - %1",getText(configFile >> "CfgVehicles" >> (typeOf _vehicle) >> "displayName")]];
 };
 
-_veh_data = [_vehicle] call life_fnc_vehicleWeight;
+private _veh_data = [_vehicle] call life_fnc_vehicleWeight;
```

Comment:
Not really a need for [] but hey ho, up to you.


Also might also be worth changing the variable name to be consistent with the rest? Preferably using camelCase. _vehData.